index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'mongo-unit' {
+    /**
+     * @returns Url to connect to the created db.
+     */
+    export function start(opts?: Partial<MongodOptions>): Promise<string>;
+
+    export function stop(): Promise<void>;
```

Comment:
`stop` returns a promise to `setTimeout`, which in turn returns timoutId.
I think the correct typing here is `Promise<number>` (correct me if I'm wrong)